javascript/javascript3/week2/Exercises.js

```diff
@@ -0,0 +1,113 @@
+async function getMovies() {
+  try {
+    const response = await fetch(
+      "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+    );
+    const movies = await response.json();
+    console.log(movies);
+    const badMovies = movies.filter((movie) => movie.rating < 6);
+    console.log(badMovies);
+    const badMoviesSince2000 = badMovies.filter((movie) => movie.year >= 2000);
+    console.log(badMoviesSince2000);
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+getMovies();
+
+// Promise that resolves after set time
+
+function resolvedAfterSetTime(resolveAfter) {
+  return new Promise(function (resolve) {
+    setTimeout(function () {
+      resolve("I am called asynchronously");
+    }, resolveAfter * 1000);
+  })
+    .then(function (result) {
+      console.log(result);
+    })
+    .catch(console.log.bind(console));
+}
+
+resolvedAfterSetTime(6);
+
+// Use it with async/await
+
+async function resolveAfterWithAsync(resolveAfter) {
+  await new Promise((resolve) => {
```

Comment:
Are you aware of how you mix anonymous functions with arrow functions?
Do you understand how they are different? 

My advice would be to pick one version and stick to it, since arrow functions are more concise and generally more predictable (with regards to context) I'd pick that style :)